<?xml version="1.0"?>
<launch>
    
    <arg name="radius" default="0.2"/>
    <arg name="robot" default="virtual_robot"/>

    <group ns="virtual_robot">
        
        <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_base_link"
              args="0 0 0 0 0 0 map base_link"/>
        
<!--        <node pkg="navfn" type="navfn_node" name="navfn" output="screen">
            
            <<rosparam file="$(find mrs_monitor)/param/costmap_common.yaml" command="load" ns="global_costmap"  subst_value="true"/>
            <rosparam file="$(find mrs_monitor)/param/costmap_common.yaml" command="load" ns="local_costmap"  subst_value="true"/>
            <rosparam file="$(find mrs_monitor)/param/local_costmap.yaml" command="load"  subst_value="true"/>
            <rosparam file="$(find mrs_monitor)/param/global_costmap.yaml" command="load" subst_value="true"/>
            <rosparam file="$(find mrs_monitor)/param/move_base.yaml" command="load" />
            <remap from="map" to="/map"/>
    </node>-->
    
    <node pkg="costmap_2d" type="costmap_2d_node" name="costmap_2d" output="screen">
        <param name="robot_base_frame" value="virtual_robot/base_link"/>
        
<!--            <rosparam file="$(find mrs_monitor)/param/costmap_common.yaml" command="load" ns="global_costmap"  subst_value="true"/>
            <rosparam file="$(find mrs_monitor)/param/global_costmap.yaml" command="load" subst_value="true"/>
            <rosparam file="$(find mrs_monitor)/param/move_base.yaml" command="load" />-->
            <remap from="map" to="/map"/>
    </node>
        
    </group>
</launch>
